internal/encoding/block/from_parquet.go

```diff
@@ -92,3 +99,49 @@ func FromParquetBy(payload []byte, partitionBy string, filter *typeof.Schema, ap
 	blocks = append(blocks, last...)
 	return blocks, nil
 }
+
+type fieldHandler func(interface{}) (interface{}, error)
+
+func getReverseHandler(typ string) (fieldHandler func(interface{}) (interface{}, error), err error) {
+
+	switch typ {
+	case "string":
+		fieldHandler = stringHandler
+	}
+
+	return fieldHandler, nil
+}
+
+// Transform runs the computed Values and overwrites/appends them to the set.
+func ApplyCastHandlers(r Row, fieldHandlers []fieldHandler) Row {
```

Comment:
This function does  not need to be exported ? Also it's not called anywhere?

Reply:
Removed, thanks